gulp/tasks/karma-fast.js

```diff
@@ -30,14 +29,15 @@ exports.task = function (done) {
 
   gutil.log('Running unit tests on unminified source.');
 
-  karma = new Server(karmaConfig, captureError(clearEnv,clearEnv));
+  var karma = new Server(karmaConfig, captureError(clearEnv,clearEnv));
```

Comment:
Is it intended to use `var` here? It kind of feels inconsistent to me (why not `const`)

Reply:
it's odd that it didn't complain.

Reply:
I updated `scripts/` but I didn't update all of the gulp tasks yet...

Reply:
probably worth running the safe transforms

Reply:
Yep, done. Pushing soon.